app/src/main/java/com/omniimpact/javaexperiments/Activities/ActivityMain.java

```diff
@@ -10,14 +10,39 @@
 
 import Fragments.FragmentMain;
 
+/**
+ * Main activity for the application where we start execution.
+ *
+ * Activities are individual "whole windows" of a Android application.
+ */
 public class ActivityMain extends AppCompatActivity {
 
+    /**
+     * Occurs on the instantiation of the application, creates the initial
+     * layout and sets it to the current content view.
+     * TODO: Add a point about how this is different from onStart()
+     * @param savedInstanceState TODO: Unknown, what is this?
+     */
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.layout_content_view_single_fragment);
     }
 
+    /**
+     * Occurs on starting the application (after instantiation). Creates
```

Comment:
application -> activity

Reply:
I changed the word application to activity.